src/Security/Group.php

```diff
@@ -721,7 +721,7 @@ public function requireDefaultRecords()
      * Code needs to be unique as it is used to identify a specific group. Ensure no duplicate
      * codes are created.
      *
-     * @deprecated 5.0 Remove deduping in favour of throwing a validation error for duplicates.
+     * @todo Replace with a validation error when a duplicate is found.
```

Comment:
Please create a card for this instead of having he todo comment.

Reply:
There's an existing issue https://github.com/silverstripe/silverstripe-framework/issues/10436 - I'll put that issue into our backlog for review